lib/internal/test_runner/test.js

```diff
@@ -806,13 +810,7 @@ class Test extends AsyncResource {
     return Number(this.endTime - this.startTime) / 1_000_000;
   }
 
-  report() {
-    countCompletedTest(this);
-    if (this.subtests.length > 0) {
-      this.reporter.plan(this.subtests[0].nesting, this.loc, this.subtests.length);
-    } else {
-      this.reportStarted();
-    }
+  #getReportDetails() {
```

Comment:
I don't know that there is a benefit to making this method private since the entire class is supposed to be hidden from users.